src/test/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModelTest.java

```diff
@@ -163,8 +163,109 @@ public void testJavadocChild() {
         Assert.assertTrue(javadoc instanceof DetailNode);
         Assert.assertEquals(JavadocTokenTypes.JAVADOC, ((DetailNode) javadoc).getType());
         final Object javadocChild = parseTree.getChild(javadoc, 2);
-        Assert.assertTrue(javadoc instanceof DetailNode);
+        Assert.assertTrue(javadocChild instanceof DetailNode);
         Assert.assertEquals(JavadocTokenTypes.TEXT, ((DetailNode) javadocChild).getType());
     }
 
+    @Test
+    public void testGetIndexOfChild() {
+        final DetailAST nChild = tree.getFirstChild().getNextSibling().getNextSibling();
+        final ParseTreeTablePModel parseTree = new ParseTreeTablePModel(null);
+        final int index = parseTree.getIndexOfChild(tree, nChild);
+        Assert.assertEquals(2, index);
+    }
+
+    @Test
+    public void testGetValueAt() {
+        final DetailAST node = tree.getFirstChild()
```

Comment:
What is this node we are getting and why this specific node and not some random other node? If test fails, we need some way to understand what went wrong, test or main code, and what expected test behavior is if input or grammar changes.

Please add some comments to describe what this node should be.
Also add an assertion to verify we are getting the correct node and not a wrong one (like node type, children, etc).